src/index.ts

```diff
@@ -1,17 +1,40 @@
 import express, { Express, Request, Response } from "express";
 import cors, { CorsRequest } from "cors";
+import passport from "passport";
+import { Strategy as JwtStrategy, ExtractJwt } from "passport-jwt";
+import helmet from "helmet";
+
 import usersRouter from "./routes/users.routes";
 import employersRouter from "./routes/employers.routes";
 import coachesRouter from "./routes/coaches.routes";
 import tagsRouter from "./routes/tags.routes";
+import { tokenRouter } from "./routes/tokens.routes";
 
 const app: Express = express();
+// app.use(helmet());
```

Comment:
Not actually using helmet yet?

Reply:
nope def should be there. Ill add it back in. Thanks for catching. Was just testing something